checked/debug.go

```diff
@@ -189,7 +193,7 @@ func (d *debugger) append(event debuggerEvent, ref int, pc []uintptr) {
 			// Shift all tracebacks back one if at end of traceback cycles
 			slice := d.entries[0]
 			for i, entry := range slice {
-				tracebackCallersPool.Put(entry.pc)
+				tracebackCallersPool.Put(&entry.pc)
```

Comment:
If you put back the address, and you nil it out immediately below, the allocated slice is lost, and when you get it back at line 297, you'll get a nil back, which probably isn't what we want.